assets/js/scripts.js

```diff
@@ -0,0 +1,92 @@
+(function () {
+
+  'use strict';
+
+  // load fonts
+  function loadFonts(href) {
+    let head = document.querySelector('head');
+    let link = document.createElement('link');
+    link.rel = 'stylesheet';
+    link.href = href;
+    head.appendChild(link);
+  }
+  loadFonts('https://fonts.googleapis.com/css?family=Montserrat:400,300,700|Work+Sans|Material+Icons&display=swap');
+  
+
+  // current years
+  function setCurrentYears(domElement) {
+    let date = new Date();
+    return domElement.textContent = date.getFullYear();
+  }
+
+  let copyrightYear = document.querySelector('.copyright-year');
+  setCurrentYears(copyrightYear);
+
+
+  // lazy load	
+  var bLazy = new Blazy({
+    // Options
+    offset: 100
```

Comment:
This offset value seems to be [the default of `bLazy`](http://dinbror.dk/blog/blazy/?ref=github).  Can we remove it?